requirements.txt

```diff
@@ -1,83 +1,61 @@
-# Core requirements
-boto3
-elasticsearch>6.0.0,<7.0.0
-elasticsearch-dsl>6.0.0,<7.0.0
-pillow
-psycopg2
-requests==2.20.0
-urllib3<1.24 # Travis problem introduced 20181016 - check to see when we can remove
-sqlalchemy==1.2.0
-flask-sqlalchemy-session
-lxml
-flask>1.0.1
-werkzeug
-isbnlib
-nose
-python-dateutil
-uwsgi
-loggly-python-handler
-mock
-py-bcrypt
-pyspellchecker
-money
-pymarc
-accept-types
```

Comment:
I don't think we use accept-types anymore. As I recall the project dropped Python 2 support and it was so small that the simplest solution was to copy it into our code base.

Reply:
That's right, but I do remember removing it from the file.